doc/api/emissions.md

```diff
@@ -0,0 +1,194 @@
+
+# Emissions domain
+
+# Get emissions
+
+The emissions api should take three query parameters
+
+## Request
+
+```text
+GET /api/emissions
+        ?dateFrom=1514826000
+        &dateTo=1514864000
+        &aggregation=TOTAL   
+```
+
+## Parameters
+
+- dateFrom: [UNIX timestamp](best-practices.md#date-from-and-to)
+- dateTo: [UNIX timestamp](best-practices.md#date-from-and-to)
+- aggregation: [aggregation ENUM](best-practices.md#aggregation)
+
+## Response
+
+```json
+{
+    "emissions": [
+        {
+            "dateFrom": 1514826000, 
+            "dateTo": 1514864000,  
+            "total": {
+                "co2": 1241245534.213   //g
+            },
+            "relative": {
+                "co2": 1234.213   // g/kWh
+            }
+        }
+    ]
+}
+```
+
+sum-agg is Sum for each "bucket" in the aggregation level selected.
```

Comment:
```suggestion
```

### Calculation

sum-agg is Sum for each "bucket" in the aggregation level selected.
```